app/templates/apis/submit-your-app.html

```diff
@@ -26,11 +26,21 @@
 
   <p>Hosting Power-Ups is the responbility of the creator. Trello does not host Power-Ups. Please ensure that you have highly available, responsible, and scalable hosting for your Power-Up.</p>
 
+  <h4>Review Process</h4>
```

Comment:
We probably should document some basic guidelines, or link out to the ones that we already have